UnitTests/ResourceManagerTests/CommitDataRenders/CommitDataHeaderRendererIntegrationTests.cs

```diff
@@ -63,8 +68,11 @@ public void Render_with_tabs_no_links()
                                  "Committer:		<a href='mailto:Jane.Doe@test.com'>Jane Doe &lt;Jane.Doe@test.com&gt;</a>" + Environment.NewLine +
                                  "Commit date:	2 days ago (" + LocalizationHelpers.GetFullDateString(_data.CommitDate) + ")" + Environment.NewLine +
                                  "Commit hash:	" + _data.Guid + Environment.NewLine +
-                                 "Children:		" + _data.ChildrenGuids[0].Substring(0, 10) + " " + _data.ChildrenGuids[1].Substring(0, 10) + " " + _data.ChildrenGuids[2].Substring(0, 10) + Environment.NewLine +
-                                 "Parent(s):		" + _data.ParentGuids[0].Substring(0, 10) + " " + _data.ParentGuids[1].Substring(0, 10);
+                                 "Children:		" + GitRevision.ToShortSha(_data.ChildrenGuids[0]) + " "
```

Comment:
Nitty picky comments on test code: It would be nice to have the same formatting for all changes. (I prefer line 50-56)

Reply:
done